code/datums/spells/wizard_spells.dm

```diff
@@ -332,6 +355,9 @@
 
 	active = FALSE
 
+/obj/effect/proc_holder/spell/fireball/apprentice
+	centcom_cancast = FALSE
```

Comment:
Why limit it from centcom for apprentices? Prevent them from hurting their master?

Reply:
Indeed, fireball can delimb so I figured it would save admins headaches to stop that interaction from happening in the first place.